app/models/reaction.rb

```diff
@@ -25,7 +25,7 @@ class Reaction < ApplicationRecord
   before_destroy :remove_algolia
 
   algoliasearch index_name: "SecuredReactions_#{Rails.env}", auto_index: false, auto_remove: false do
-    attribute :id, :reactable_user, :searchable_reactable_title, :searchable_reactable_path, :status,
+    attribute :id, :reactable_user, :searchable_reactable_title, :searchable_reactable_path, :status, :reading_time,
               :searchable_reactable_text, :searchable_reactable_tags, :viewable_by, :reactable_tags, :reactable_published_date
     searchableAttributes %i[searchable_reactable_title searchable_reactable_text
                             searchable_reactable_text searchable_reactable_tags reactable_user]
```

Comment:
It's not related to code you wrote but I noticed `searchable_reactable_text` appears twice.

Reply:
Nice catch.

Reply:
Just removed the duplicate. Thanks for pointing this out, @bolariin! 😄